pages/who-we-are-hiring.md

```diff
@@ -15,20 +13,59 @@ We’re building a team that looks like the United States, and we don't discrimi
 
 Everywhere in the United States! Most of our team is distributed across the country in places like Chicago, New York, Raleigh, Tucson, Austin, Dayton, Philadelphia, San Diego, Seattle, and Portland. [Read more about how our work culture supports distributed teams.](https://18f.gsa.gov/2015/10/15/best-practices-for-distributed-teams/) If you have no experience working on remote teams or are applying for a role that requires in-person interactions with clients, we'll ask you to work out of our offices in San Francisco or D.C.
 
-## Upcoming positions
-We're currently revamping our application process. When it relaunches in mid-November, we will be hiring for positions **including but not limited to** the following:
+## 18F Roles
```

Comment:
@rooneywp Understood, thanks for the explanation. I'd kind of like it to be "Roles at 18F" but I won't hold up the merge for that. :)

We should keep the "We're currently revamping our application process. When it relaunches in mid-November, we will be hiring for positions **including but not limited to** the following:" until the new application goes live on Monday, yes?